### ISSUE TITLE
CSA can search from chat history

### ISSUE THREAD

**Issue by janinakimtrohlev** [buerokratt/Buerokratt-Chatbot#122]
AS A CSA, 
I WANT TO be able to search from chat history based on an entered string; results are filtered concurrently in the history table with an option to reset search.
SO THAT helps me to find faster

### Acceptance criteria

- [x] String searches from chat history, inside messages
- [x] Search filter can be reset
- [x] Table is automatically updated with found matches

#### Integration

- [ ] _Previously_ used endpoint `/cs-get-chat-ids-matching-message-search`
- [x] FYI - related to #19

**Comment by PaulaMerle** [comment]
Additional search options are covered with #19

**Comment by turnerrainer** [comment]
@joonasroosalung do we have an endpoint for this one?

**Comment by 1AhmedYasser** [comment]
@rasmusei Can't seem to find this endpoint in the codebase for chat bot `/cs-get-chat-ids-matching-message-search`

**Comment by rasmusei** [comment]
@turnerrainer this is for you

**Comment by ffrose** [comment]
Once End-User messages arrive to CSA we are able to start testing this task. https://github.com/buerokratt/Buerokratt-Chatbot/issues/250#top

**Comment by turnerrainer** [comment]
> Once End-User messages arrive to CSA we are able to start testing this task. [#250 (comment)](https://github.com/buerokratt/Buerokratt-Chatbot/issues/250#top)

@ffrose is this still blocked?

**Comment by ffrose** [comment]
> > Once End-User messages arrive to CSA we are able to start testing this task. [#250 (comment)](https://github.com/buerokratt/Buerokratt-Chatbot/issues/250#top)
> 
> @ffrose is this still blocked?

@turnerrainer No, this is not blocked anymore. I am able to start testing this task.

**Comment by ffrose** [comment]
BUG - you are able to search words inside messages but cannot search conversations using any filters (for example: chat ID)
![image](https://github.com/buerokratt/Buerokratt-Chatbot/assets/119657383/8aa51e36-03a1-487d-9b38-32a0256e610c)

**Comment by 1AhmedYasser** [comment]
> BUG - you are able to search words inside messages but cannot search conversations using any filters (for example: chat ID) ![image](https://user-images.githubusercontent.com/119657383/258112421-8aa51e36-03a1-487d-9b38-32a0256e610c.png)

Fixed in https://github.com/buerokratt/Buerokratt-Chatbot/pull/264

**Comment by MarkusVeltri** [comment]
Working as intended.
